config/config/common-config.yaml

```diff
@@ -572,6 +579,10 @@ ingressNginx:
       ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
       annotations: {}
 
+      ## Enable node port allocation
+      ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-nodeport-allocation
+      allocateLoadBalancerNodePorts: true
```

Comment:
could you open an issue to check if thsi is needed for all providers?

Reply:
Issue created here: https://github.com/elastisys/compliantkubernetes-apps/issues/2002